### PR TITLE
Re-enable toes 

### PR DESCRIPTION
**Pull request by yaserOSource**
This PR will re-enable toes for avatars, I also added toes in the leg manager but did not expand on the existing foot transforms such as hmd_floor as I thought maybe this might be already in use.

### CODE REVIEW

File: avatars/vrarmik/LegsManager.js [hunk 1]
```diff
@@ -54,6 +54,8 @@ class Leg {
     this.foot.endTransform = new THREE.Object3D();
     this.foot.startHmdFloorTransform = new THREE.Object3D();
 
+    this.toe = new THREE.Object3D();
```
Comment (avaer) [inline avatars/vrarmik/LegsManager.js:57]: Doesn't this toe need to be connected up to rotate correctly? See the lines below.

File: avatars/vrarmik/LegsManager.js [hunk 2]
```diff
@@ -146,6 +149,10 @@ class Leg {
         .multiply(downHalfRotation)
         .premultiply(Helpers.getWorldQuaternion(this.lowerLeg, localQuaternion2).invert());
       Helpers.updateMatrixMatrixWorld(this.foot);
+
+
+      this.toe.quaternion.copy(footRotation).multiply(downHalfRotation).premultiply(Helpers.getWorldQuaternion(this.lowerLeg, localQuaternion2).invert());
```
Comment (avaer) [inline avatars/vrarmik/LegsManager.js:154]: I'm not sure if this is correct; we should test it by hacking IK to be enabled: https://github.com/webaverse/app/blob/5af9f875958852454c83bee49f18cfcbb044ac49/avatars/avatars.js#L2156

**Comment by yaserOSource** [comment]
https://user-images.githubusercontent.com/51108458/140411074-953ffacf-e877-4ba5-b9bd-3f017a697ff4.mp4

This is the toe bones enabled with IK

**Comment by yaserOSource** [comment]
https://user-images.githubusercontent.com/51108458/140411159-f74c4e85-0a74-4a4b-a8a6-adf863ca3a16.mp4
This is toe bones enabled without IK

**Review by avaer — APPROVED** [review]
LGTM.

**Comment by avaer** [comment]
Btw @yaserOSource if you put "Fixes https://github.com/webaverse/app/issues/1663" in the topic it will auto close on merge.

**Comment by avaer** [comment]
Fixes https://github.com/webaverse/app/issues/1663

**Pull request merged**